indico/modules/events/papers/schemas.py

```diff
@@ -192,6 +194,13 @@ class PaperSchema(mm.Schema):
     can_judge = Function(lambda paper, ctx: paper.can_judge(ctx.get('user')))
     can_comment = Function(lambda paper, ctx: paper.can_comment(ctx.get('user'), check_state=True))
     can_review = Function(lambda paper, ctx: paper.can_review(ctx.get('user')))
+    can_submit_proceedings = Function(lambda paper, ctx: paper.contribution.can_submit_proceedings(ctx.get('user')))
+    editing_open = Function(
+        lambda paper, ctx: editable_type_settings[EditableType.paper].get(paper.event, 'submission_enabled')
+    )
+    editing_enabled = Function(
+        lambda paper, ctx: 'paper' in editing_settings.get(paper.event, 'editable_types')
+    )
```

Comment:
What if you enabled paper editing but then disabled the Editing feature altogether? I think you also need a check for `paper.event.has_feature('editing')`

Reply:
I can add that, but I don't think it will affect this PR because the call to `file-types` returns 404 if the editing feature is disabled, no file types. I'm using that as the third condition to disable it.

Reply:
feels a bit implicit even if it works... and `editing_enabled: true` would be incorrect if editing isn't enabled ;)